migrations/m161204_184730_add_position_column_to_orders_table.php

```diff
@@ -12,7 +12,6 @@ class m161204_184730_add_position_column_to_orders_table extends Migration
      */
     public function up()
     {
-        $this->addColumn('orders', 'buyer_id', $this->int()->notNull());
```

Comment:
It's better to add all the fields in one query (with one line of code)
Your code adds the columns one-by-one, it's not the optimal way. Please write one operator for creating the whole table next time.